### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by FloncDev**
Solves #7

### CODE REVIEW

File: src/frontend/github-markdown.css
Comment (FloncDev) [inline src/frontend/github-markdown.css:1]: Not my css, took it from somewhere else but was getting some errors when using their link.

**Comment by iauu** [comment]
Working for me, Just the page is slightly larger which still requires scrolling on my computer

**Comment by iauu** [comment]
Also can be changing the rendering in a function to make the code look cleaner I think

**Review by iauu — APPROVED** [review]
Working code, look nices. Require future modification on other platform but not now.

**Pull request merged**